wgsl/index.bs

```diff
@@ -4033,82 +4033,158 @@ WGSL [=predeclared|predeclares=] an [=enumerant=] for each of the texel formats
 
 ### Sampled Texture Types ### {#sampled-texture-type}
 
-<pre class='def'>
-`texture_1d<type>`
-`texture_2d<type>`
-`texture_2d_array<type>`
-`texture_3d<type>`
-`texture_cube<type>`
-`texture_cube_array<type>`
-</pre>
-* `type` [=shader-creation error|must=] be `f32`, `i32`, or `u32`
+A <dfn noexport dfn-for='type'>sampled texture</dfn> is capable
+of being [=memory access|accessed=] in conjunction with a [=type/sampler=].
+It can also be accessed without the use of a sampler.
+Sampled textures only allow [=read accesses=].
+
+The [=texel format=] of the texture is defined in the {{GPUTextureBindingLayout}}.
+
+<table class='data'>
+  <thead>
+    <tr><th>Type<th>[=texture/Dimensionality=]<th>[=texture/Arrayed=]
+  </thead>
+  <tr><td><dfn noexport dfn-for='type'>texture_1d</dfn><*T*>
+      <td>1D
+      <td>No
+  <tr><td><dfn noexport dfn-for='type'>texture_2d</dfn><*T*>
+      <td>2D
+      <td>No
+  <tr><td><dfn noexport dfn-for='type'>texture_2d_array</dfn><*T*>
+      <td>2D
+      <td>Yes
+  <tr><td><dfn noexport dfn-for='type'>texture_3d</dfn><*T*>
+      <td>3D
+      <td>No
+  <tr><td><dfn noexport dfn-for='type'>texture_cube</dfn><*T*>
+      <td>Cube
+      <td>No
+  <tr><td><dfn noexport dfn-for='type'>texture_cube_array</dfn><*T*>
+      <td>Cube
+      <td>Yes
+</table>
+
+* *T* [=shader-creation error|must=] be `f32`, `i32`, or `u32`
```

Comment:
Should explicitly define T as the 'sampled type' for the texture.

That's the terminology used in the API spec for validating the binding layout.
See uses of 'sampled type' at https://gpuweb.github.io/gpuweb/#abstract-opdef-validating-shader-binding 

(Something in me wants to call it the 'sampled component type', but that breaks with tradition)